src/goto-symex/path_storage.h

```diff
@@ -90,6 +90,13 @@ class path_storaget
   /// Counter for nondet objects, which require unique names
   symex_nondet_generatort build_symex_nondet;
 
+  using global_name_mapt = std::map<irep_idt, std::pair<ssa_exprt, unsigned>>;
```

Comment:
Just a comment about the commit message: please wrap lines at 80 or 72 characters, otherwise `git log` becomes unusable.